src/codegen/compile.ml

```diff
@@ -4108,11 +4109,11 @@ module IC = struct
     assert (E.mode env = Flags.ICMode || E.mode env = Flags.RefMode);
     let empty_f = Func.of_body env [] [] (fun env ->
       Lifecycle.trans env Lifecycle.InInit ^^
-
```

Comment:
@crusso I also tried putting the call to `region_init` here.  The error indicates that something else is wrong, not the placement of the call, I think?

```
+OOPS! You've triggered a compiler bug.
+Please report this at https://github.com/dfinity/motoko/issues/new with the following details:
+
+Motoko (source XXX)
+
+Fatal error: exception Lib.Promise.Promise
Some tests failed:
stable-region-blob.mo stable-region-blob.mo
```

Maybe I'm not doing the import the right way, somehow?

Reply:
Yeah, this is probably the wrong place. Can you put it in the "init" functions, probably right after destabilization?